VARIABLES.md

```diff
@@ -598,15 +622,15 @@ Default:  "{{ schema_registry.properties }}"
 
 ### kafka_rest_user
 
-Only use to customize Linux User Rest Proxy Service runs with. User must exist on host.
+Only use to customize Linux User Rest Proxy Service runs with. Default user is cp-kafka-rest
```

Comment:
Modify this variable to customize the Linux User that the Rest Proxy Service runs with. Default user is cp-kafka-rest.